test/parallel/test-tick-processor.js

```diff
@@ -18,7 +19,7 @@ common.refreshTmpDir();
 process.chdir(common.tmpDir);
 // Unknown checked for to prevent flakiness, if pattern is not found,
 // then a large number of unknown ticks should be present
-runTest(/LazyCompile.*\[eval\]:1|.*%  UNKNOWN/,
+runTest(/LazyCompile.*test.js:1|.*%  UNKNOWN/,
```

Comment:
Urgh, so maybe the reason this makes it reliable is because the name of the file is always going to be there. I guess we'll see when the change is made.

Regardless, might be a good idea to preserve all this information-sharing in a comment or seven in the file.